### PR TITLE
Fix: some tips w/o images not visualized

### PR DESCRIPTION
**Pull request by mradkov**
Fix: some tips without images, but with valid description and title not being visualized.

- update isPreviewToBeVisualized check

### CODE REVIEW

**Comment by thepiwo** [comment]
can you refer the corresponding issue?

File: src/components/tipRecords/TipRecord.vue [hunk 1]
```diff
@@ -98,13 +98,13 @@ export default {
       return this.tip.preview.responseUrl ? new URL(this.tip.preview.responseUrl).hostname : '';
     },
     tipPreviewImage() {
-      return this.isPreviewToBeVisualized(this.tip) ? Backend.getTipPreviewUrl(this.tip.preview.image) : '';
+      return this.isPreviewToBeVisualized(this.tip) && this.tip.preview.image !== null ? Backend.getTipPreviewUrl(this.tip.preview.image) : '';
```
Comment (venimus) [inline src/components/tipRecords/TipRecord.vue]: long line

File: src/components/tipRecords/TipRecord.vue [hunk 2]
```diff
@@ -98,13 +98,13 @@ export default {
...
     },
   },
   methods: {
     isPreviewToBeVisualized(tip) {
       return typeof tip !== 'undefined' && tip !== null
-          && typeof tip.preview !== 'undefined' && tip.preview.image !== null
+          && typeof tip.preview !== 'undefined'
```
Comment (thepiwo) [inline src/components/tipRecords/TipRecord.vue]: why does it make a change where this check is defined?

Comment (venimus) [reply src/components/tipRecords/TipRecord.vue]: I guess preview could be without image?

Comment (thepiwo) [reply src/components/tipRecords/TipRecord.vue]: I see

Comment (mradkov) [reply src/components/tipRecords/TipRecord.vue]: This prevents the fallback image from showing.

**Pull request merged**